src/backend/ipcHandlers/mods.ts

```diff
@@ -0,0 +1,306 @@
+import { notify, showDialogBoxModalAuto } from 'backend/dialog/dialog'
+import { cancelQueueExtraction } from 'backend/downloadmanager/downloadqueue'
+import { LogPrefix, logDebug, logError, logInfo } from 'backend/logger/logger'
+import { getMainWindow, sendFrontendMessage } from 'backend/main_window'
+import {
+  ExtractZipProgressResponse,
+  ExtractZipService
+} from 'backend/services/ExtractZipService'
+import { libraryManagerMap } from 'backend/storeManagers'
+import {
+  calculateProgress,
+  getDestinationPath,
+  inProgressExtractionsMap
+} from 'backend/storeManagers/hyperplay/games'
+import { copyRecursiveSync } from 'backend/utils'
+import { callAbortController } from 'backend/utils/aborthandler/aborthandler'
+import { readdirSync, rm } from 'graceful-fs'
+
+import i18next from 'i18next'
+import path from 'path'
+
+export async function prepareBaseGameForModding({
+  appName,
+  zipFile,
+  installPath
+}: {
+  appName: string
+  zipFile: string
+  installPath: string
+}) {
+  const window = getMainWindow()
+  if (!window) return
```

Comment:
We should add a `console.warn` of the early return.

I've done this and then spend a lot time debugging why the action is doing nothing.

Reply:
Doesn't a `{}` around the `if` fix it? that is why I always use it. 
Although, `window` will always be defined here since the user needed to have a window to start the installation.